src/md/src/WorkWithReports/ViewLaunches.md

```diff
@@ -256,3 +259,24 @@ You can navigate from one log to the other using "Next"/"Previous" buttons on th
 
 
 Navigation takes into account selected by user filter and hidden preconditions methods on Step view.
+
+### Retried test case (retry)
+
+In case you implement a retry logic for your tests in a test framework, the ReportPortal will reflect them as a retry test case. If there were a few invocations of the one test case, all these invocations will be shown as the one test case in the ReportPortal.
```

Comment:
In case you implement a retry logic for your tests in a test framework, ReportPortal will reflect it by adding special retry structure. If there were a few invocations of the one test case, all these invocations will be shown as the one test case in ReportPortal.